packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -28,6 +28,7 @@ export interface CommerceApiProviderProps extends ApiClientConfigParams {
     locale: string
     currency: string
     queryClientConfig?: QueryClientConfig
+    correlationId?: string
```

Comment:
```suggestion
    headers?: Record<string, string>
```

[suggestion](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/709/files/d8f5ef429e4aae73d1af97cebb0cbe037388da3b#r971208788)